scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -227,7 +227,10 @@ object BigQueryTable {
     private[bigquery] val DefaultTimePartitioning: TimePartitioning = null
     private[bigquery] val DefaultExtendedErrorInfo: ExtendedErrorInfo = ExtendedErrorInfo.Disabled
     private[bigquery] val DefaultInsertErrorTransform
-      : SCollection[DefaultExtendedErrorInfo.Info] => Unit = _ => ()
+      : SCollection[DefaultExtendedErrorInfo.Info] => Unit =
+      // A NoOp on the extended error output, so that we don't have DropInputs
+      // in the pipeline graph.
+      _.withName("Ignore ExtendedErrorInfo").map(_ => ())
```

Comment:
@anish749 
This is actually independent of the `ExtendedErrorInfo` value so no need to.
> Set this if ExtendedErrorInfo is disabled, else leave it as is.

I would also rename as:

```suggestion
      _.withName("DropFailedInserts").map(_ => ())
```

there's another place where this is required.